topics/ecology/tutorials/x-array-map-plot/tutorial.md

```diff
@@ -0,0 +1,461 @@
+>
+>![ECMWF Reanalysis Air temperature a 2 metres on 2022-05-25 at 17:00:00 UTC](../../images/ECMWF_Reanalysis_17.png)
+>
+>    > ### {% icon comment %} Why sometimes we shift longitudes ?
+>    >
+>    > Longitudes are measured in degrees from 0 to 360. We need to change longitudes so that `NetCDF xarray map plotting` can properly plot our dataset because we don't have worldwide data but only cover Europe.
+>{: .comment}
+>  
+{: .hands_on}
+
+
+
+> ### {% icon question %} Questions
+>
+> 1. What are the different kinds of projections that can be used?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1.There are many projections which can be used in the `NetCDF xarray map plotting` tool. Find here a list of valid projections. Different projections have different purposes and need to be carefully chosen. Follow this sheet to [find the syntax for projections](https://github.com/Quickbeasts51429/xarray_projection). 
+> >
+> >## Plotting different projections using **NetCDF xarray map plotting**
+ > > > ### {% icon hands_on %} Hands-on: Plotting the major projections using the xarray tool.
+> > >
+> > > 1. {% tool [NetCDF xarray map plotting](toolshed.g2.bx.psu.edu/repos/ecology/xarray_mapplot/xarray_mapplot/0.20.2+galaxy0) %} with the following parameters:
+> > >    - {% icon param-file %}
+*"Netcdf file"*: `outfile_18.nc`
+> > >    - {% icon param-file %} *"Tabular of variables"*: `Metadata infos from outfile_18.nc` (output of **NetCDF xarray Metadata Info** {% icon tool %})
+> > >    - *"Choose the variable to plot"*: `air_temperature_at_2_metres`
+> > >    - *"Name of latitude coordinate"*: `lat`
+> > >    - *"Name of longitude coordinate"*: `lon`
+> > >    - *"Datetime selection"*: `No`
+> > >    - *"Range of values for plotting e.g. minimum value and maximum value (minval,maxval) (optional)"*: `220,320`
+> > >    - *"Add country borders with alpha value [0-1] (optional)"*: `1.0`
+> > >    - *"Add coastline with alpha value [0-1] (optional)"*: `1.0`
+> > >    - *"Add ocean with alpha value [0-1] (optional)"*: `1.0`
+> > >    - *"Specify plot title (optional)"*: `Projection :  Mercator 18:00 UTC `
+> > >    - *"Specify which colormap to use for plotting (optional)"*: `lajolla`
+> > >    - *"Specify the projection (proj4) on which we draw e.g. {"proj":"PlateCarree"} with double quote (optional)"*: `{'proj': 'Mercator'}`
+> > >
+> > >    ![ECMWF Reanalysis Air temperature a 2 metres on 2022-05-25 at 18:00:00 : CMRmap](../../images/projselect.png)
+> > >
+> > >The final plot is shown below:
+> > >
+> > >    ![ECMWF Reanalysis Air temperature a 2 metres on 2022-05-25 >at 18:00:00](../../images/Mercator.png)
+> > >
+> > >
+> > >
+> > >Some other important types of projections can be found below :
+> > >
+> > > - *{"proj":"InterruptedGoodeHomolosine" }*
+> > >![ECMWF Reanalysis Air temperature a 2 metres on 2022-05-25 >at 18:00:00](../../images/InterruptedGoodeHomolosine.png)
```

Comment:
I did not manage to reproduce this plot due to "malformed node or string" somewhere

Reply:
Do you mean , {"proj":"InterruptedGoodeHomolosine" } doesnot produce a plot?